website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -153,6 +153,19 @@ A `agent_pool_profile` block supports the following:
 
 * `count` - (Required) Number of Agents (VMs) in the Pool. Possible values must be in the range of 1 to 100 (inclusive). Defaults to `1`.
 
+~> **NOTE:** When `enable_auto_scaling` parameter is set to true, this value will dynamically change depending on the cluster's load. It is recommended to ignore changes to it with autoscaling enabled, since autoscaled clusters cannot be scaled manually.  This can be done by adding following lifecycle hook to the resource for each agent pool profile with autoscaling enabled.
```

Comment:
Sorry, but this does not really look like a solution to me. To be honest haven't checked Kubernetes Cluster creation code yet, but i would expect something like if enable_auto_scaling and  is not NewResource skip touching count.